IbisLib/polydataobject.h

```diff
@@ -19,6 +19,8 @@ See Copyright.txt or http://ibisneuronav.org/Copyright.html for details.
 #include <vtkProperty.h>
 #include <vtkSmartPointer.h>
 
+#include "abstractpolydataobject.h"
+
 class vtkPolyData;
```

Comment:
vtkPolyData, vtkTransform, vtkActor, vtkProbeFilter, vtkScalarsToColors, ImageObject, vtkClipPolyData, vtkPassThrough, vtkCutter. vtkPlane and vtkPlanes are not used in polydataobject.h